Simplenote.xcodeproj/project.pbxproj

```diff
@@ -22,7 +22,7 @@
 			isa = PBXAggregateTarget;
 			buildConfigurationList = B5C7BF98230C59F5000DEC91 /* Build configuration list for PBXAggregateTarget "SimplenoteSecrets" */;
 			buildPhases = (
-				B5C7BF9E230C5A79000DEC91 /* ShellScript */,
+				B5C7BF9E230C5A79000DEC91 /* Run Script */,
```

Comment:
Can you believe that the Xcode UI shows "Run Script" in the name of the build phase by default, but, until you actually go in and touch the string, doesn't set the value in here?

Maybe that was a quirk of the version in which the aggregate target for the secrets was added, but still...